src/tiledb/cloud/vector_search/file_ingestion.py

```diff
@@ -0,0 +1,355 @@
+from typing import Dict, List, Mapping, Optional, Sequence
+
+from tiledb.cloud import dag
+from tiledb.cloud.utilities import as_batch
+
+
+def ingest_files(
+    file_dir_uri: str,
+    index_uri: str,
+    file_name: Optional[str] = None,
```

Comment:
This along with include/exclude don't make sense. How is this to work with TileDB files? There is no check if the TileDB file name or any parsing of the TileDB URIs. The goal again as outlined in the requirements is to use this with TileDB files, either standalone or from a group.

Reply:
Removed the `file_name` option. FileStore URIs work directly using the `search_uri` param.